<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
   <class>FileSharingDialog</class>
   <widget class="QWidget" name="FileSharingDialog">
      <property name="geometry">
         <rect>
            <x>0</x>
            <y>0</y>
            <width>978</width>
            <height>841</height>
         </rect>
      </property>
      <property name="minimumSize">
         <size>
            <width>960</width>
            <height>682</height>
         </size>
      </property>
      <property name="windowTitle">
         <string>Form</string>
      </property>
      <layout class="QVBoxLayout" name="topLayout" stretch="1">
         <property name="rightMargin">
            <number>9</number>
         </property>
         <item>
            <widget class="QFrame" name="frame_BG">
               <property name="minimumSize">
                  <size>
                     <width>850</width>
                     <height>610</height>
                  </size>
               </property>
               <property name="maximumSize">
                  <size>
                     <width>16777215</width>
                     <height>16777215</height>
                  </size>
               </property>
               <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1">
                  <property name="leftMargin">
                     <number>0</number>
                  </property>
                  <property name="topMargin">
                     <number>0</number>
                  </property>
                  <property name="rightMargin">
                     <number>0</number>
                  </property>
                  <property name="bottomMargin">
                     <number>6</number>
                  </property>
                  <item>
                     <layout class="QVBoxLayout" name="verticalLayout_Master" stretch="0,1">
                        <property name="leftMargin">
                           <number>0</number>
                        </property>
                        <property name="topMargin">
                           <number>0</number>
                        </property>
                        <property name="rightMargin">
                           <number>0</number>
                        </property>
                        <property name="bottomMargin">
                           <number>0</number>
                        </property>
                        <!-- <item>
                           <widget class="QFrame" name="frame_Header">
                              <layout class="QVBoxLayout" name="verticalLayout_Header" stretch="0">
                                 <property name="leftMargin">
                                    <number>0</number>
                                 </property>
                                 <property name="topMargin">
                                    <number>0</number>
                                 </property>
                                 <property name="rightMargin">
                                    <number>0</number>
                                 </property>
                                 <property name="bottomMargin">
                                    <number>0</number>
                                 </property>
                                 <item>
                                    <layout class="QHBoxLayout" name="horizontalLayout_Header" stretch="1,0,1">
                                       <property name="spacing">
                                          <number>6</number>
                                       </property>
                                       <item>
                                          <widget class="QLabel" name="labelFileSharingHeaderLeft">
                                             <property name="minimumSize">
                                                <size>
                                                   <width>464</width>
                                                   <height>60</height>
                                                </size>
                                             </property>
                                             <property name="maximumSize">
                                                <size>
                                                   <width>16777215</width>
                                                   <height>60</height>
                                                </size>
                                             </property>
                                             <property name="font">
                                                <font>
                                                   <pointsize>20</pointsize>
                                                   <weight>75</weight>
                                                   <bold>true</bold>
                                                </font>
                                             </property>
                                             <property name="text">
                                                <string>FILE SHARING</string>
                                             </property>
                                             <property name="alignment">
                                                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                                             </property>
                                          </widget>
                                       </item>
                                       <item>
                                          <spacer name="horizontalSpacer_Header">
                                             <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                             </property>
                                             <property name="sizeHint" stdset="0">
                                                <size>
                                                   <width>40</width>
                                                   <height>20</height>
                                                </size>
                                             </property>
                                          </spacer>
                                       </item>
                                       <item>
                                          <widget class="QLabel" name="labelFileSharingHeaderRight">
                                             <property name="minimumSize">
                                                <size>
                                                   <width>464</width>
                                                   <height>60</height>
                                                </size>
                                             </property>
                                             <property name="maximumSize">
                                                <size>
                                                   <width>16777215</width>
                                                   <height>60</height>
                                                </size>
                                             </property>
                                             <property name="font">
                                                <font>
                                                   <pointsize>14</pointsize>
                                                </font>
                                             </property>
                                             <property name="text">
                                                <string />
                                             </property>
                                             <property name="alignment">
                                                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                             </property>
                                          </widget>
                                       </item>
                                    </layout>
                                 </item>
                              </layout>
                           </widget>
                        </item> -->
                        <item>
                           <widget class="QFrame" name="frame_Content">
                              <layout class="QVBoxLayout" name="verticalLayout">
                                 <item>
                                    <widget class="QTabBar" name="tabBar" native="true" />
                                 </item>
                                 <item>
                                    <widget class="QStackedWidget" name="stackedWidget">
                                       <property name="currentIndex">
                                          <number>0</number>
                                       </property>
                                       <widget class="QWidget" name="page">
                                          <layout class="QVBoxLayout" name="verticalLayout_2">
                                             <item>
                                                <widget class="ConnectionWidget" name="connectionWidget" native="true" />
                                             </item>
                                          </layout>
                                       </widget>
                                       <widget class="QWidget" name="page_2">
                                          <layout class="QVBoxLayout" name="verticalLayout_3">
                                             <item>
                                                <widget class="ServerWidget" name="serverWidget" native="true" />
                                             </item>
                                          </layout>
                                       </widget>
                                    </widget>
                                 </item>
                              </layout>
                           </widget>
                        </item>
                     </layout>
                  </item>
               </layout>
            </widget>
         </item>
      </layout>
   </widget>
   <customwidgets>
      <customwidget>
         <class>QTabBar</class>
         <extends>QWidget</extends>
         <header>QTabBar</header>
         <container>1</container>
      </customwidget>
      <customwidget>
         <class>ConnectionWidget</class>
         <extends>QWidget</extends>
         <header>connectionwidget.h</header>
         <container>1</container>
      </customwidget>
      <customwidget>
         <class>ServerWidget</class>
         <extends>QWidget</extends>
         <header>serverwidget.h</header>
         <container>1</container>
      </customwidget>
   </customwidgets>
   <resources />
   <connections />
</ui>